config/initializers/s3.rb

```diff
@@ -0,0 +1,12 @@
+# Make sure these variables are set in .bash_profile or something.
+# Can later use Heroku to set ENV vars.  See https://devcenter.heroku.com/articles/config-vars
+
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    :provider               => 'AWS',
+    :aws_access_key_id      => ENV['S3_KEY'],
+    :aws_secret_access_key  => ENV['S3_SECRET'],
+    :region                 => ENV['S3_REGION'] # Change this for different AWS region. Default is 'us-east-1'
```

Comment:
Line is too long. [110/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.